src/js/program/program-listeners.js

```diff
@@ -27,10 +27,13 @@ export function ProviderListener(mediaController) {
                     mediaController.thenPlayPromise.cancel();
                     mediaModel.srcReset();
                 }
-                // Always fire change:mediaState to keep player model in sync
-                const previousState = mediaModel.attributes.mediaState;
-                mediaModel.attributes.mediaState = data.newstate;
-                mediaModel.trigger('change:mediaState', mediaModel, data.newstate, previousState);
+
+                if (mediaController.attached && !mediaController.background) {
```

Comment:
Backgrounded media controllers don't emit events while backgrounded. Is this being queued?

Only the state of the mediaModel after reattaching should come out. I don't think this is the right place to address this. I would need to see the callstack at the point the pause event is being fired to determine the best course of action.

Reply:
Can we try replacing the trigger work-around with `mediaModel.set('mediaState', data.newstate);` ? That should only fire on change. We may need to do a little more to queue or ignore changes while detached.

Reply:
scratch that.

Since this involves instream mode, it should be handled higher up in the controller:
```js
_model.on('change:mediaModel', function(model, mediaModel) {
            model.set('errorEvent', undefined);
            mediaModel.change('mediaState', function (changedMediaModel, state) {
                if (!model.get('errorEvent') && !model.get('instream')) {
                    model.set(PLAYER_STATE, normalizeState(state));
                }
            }, this);
```